Cargo.toml

```diff
@@ -36,6 +36,7 @@ linux-headers = [ "webkit2gtk/v2_36" ]
 [dependencies]
 libc = "0.2"
 log = "0.4"
+tracing = "0.1"
```

Comment:
should we add this behind a feature flag? so ultimately tauri uses could opt-into disabling the tracing if they wish so